src/rules/react.ts

```diff
@@ -20,6 +22,8 @@ export default {
   'jsx-no-string-ref': true,
   // Enforces that JSX elements with no children are self-closing.
   'jsx-self-close': true,
+  // Enforces use of a translation function. Plain string literals are disallowed in JSX when enabled.
+  'jsx-use-translation-function': true,
```

Comment:
💯 Might take a while to get Neutron working with this, but our users will love us for it.